backend/functions/orders-suggestions/orders-suggestions.function.test.ts

```diff
@@ -1,8 +1,8 @@
 import httpTrigger from './orders-suggestions.function';
 import { CASE_SUMMARIES } from '../lib/testing/mock-data/case-summaries.mock';
 import { CamsError } from '../lib/common-errors/cams-error';
-
-const context = require('azure-function-context-mock');
+import { createMockApplicationContext } from '../lib/testing/testing-utilities';
+import { ApplicationContext } from '../lib/adapters/types/basic';
```

Comment:
The interesting thing with this is that the function definitions were the only place where the `azure-function-context-mock` should have been used previously. Now we are providing an `ApplicationContext` to the functions where a `Context` is all that is needed. This probably won't hurt anything since we're provided a superset of properties, but it's definitely not the object the Function is expecting.